sdk/typescript/standalone/.gitignore

```diff
@@ -0,0 +1,4 @@
+/dagger.gen.go
+/internal/dagger
+/internal/querybuilder
+/internal/telemetry
```

Comment:
Why the files in `internal` are pushed if they are ignored by this files?

Reply:
I probably added them before adding the .gitignore. I'm gonna leave them in so we don't have weird linter complaints.

Reply:
Okay!